README.md

```diff
@@ -11,7 +11,7 @@ permitting tools like Zotero (http://www.zotero.org) to grab citations.
 License
 -------
 This plugin is licensed under the GNU General Public License v2. See the
-accompanying OJS file docs/COPYING for the complete terms of this license.
+accompanying OJS file `docs/COPYING` for the complete terms of this license.
```

Comment:
Actually, the license is in the file `LICENSE` in this repository; can you update this to refer to that instead?